### ISSUE TITLE
Unexpected width value in .user-page .provider .favicon CSS width

### ISSUE THREAD

**Issue by pdehaan** [mozilla/chronicle#311]
In [/app/styles/modules/_user_page.scss:35](https://github.com/mozilla/chronicle/blob/408fdde81b084704b5c5c82c6f9506bda7b58d1e/app/styles/modules/_user_page.scss#L35) we have the following code:
```scss
    .favicon {
      height: $half-grid;
      margin-right: 5px;
      vertical-align: -3px;
      width: 1$half-grid; /* <---- this guy! */
    }
```

Which compiles to this [de-minified] code:
```css
    .user-page .provider .favicon {
      height: 16px;
      margin-right: 5px;
      vertical-align: -3px;
      width: 1 16px; /* <---- this guy! */
    }
```

CSSLint blows up with this hawkward error:
> _**WARNING:** Expected end of value but found '16px'. Properties should be known (listed in CSS3 specification) or be a vendor-prefixed property. (known-properties) Browsers: All_

**Comment by pdehaan** [comment]
OK, after messing around with setting up grunt-scsslint and grunt-contrib-csslint locally, I got it down to the following warnings and errors. I can probably convert these into a PR and we can run them manually whenever we feel the urge to audit Sass/CSS:

```sh
$ grunt scsslint --force
Running "scsslint:public" (scsslint) task
File /Users/pdehaan/dev/github/chronicle/app/styles/_modules.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/main.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/_base.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/_mixins.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/_variables.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/modules/_actions.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/_layout.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/_state.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/modules/_global_footer.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/modules/_row.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/modules/_search_result.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/modules/_global_header.scss lint free
File /Users/pdehaan/dev/github/chronicle/app/styles/modules/_user_info.scss lint free
/Users/pdehaan/dev/github/chronicle/app/styles/modules/_user_page.scss:112 [W] Merge rule `&.small + &.small` with rule on line 90
Warning: Exited with error code 65 Used --force, continuing.

Done, but with warnings.

$ grunt csslint
Running "csslint:styles" (csslint) task
Linting public/styles/compiled.css...ERROR
[L63:C9]
WARNING: Expected (start | end | center | justify) but found 'distribute'. Properties should be known (listed in CSS3 specification) or be a vendor-prefixed property. (known-properties) Browsers: All
[L243:C7]
WARNING: Expected end of value but found '16px'. Properties should be known (listed in CSS3 specification) or be a vendor-prefixed property. (known-properties) Browsers: All
>> 1 file lint free.

Done, without errors.
```

The warning from scss-lint is silly (but a good rule).
The warnings from csslint aren't great. The first one just seems like a [valid IE 10 issue](http://realworldvalidator.com/css/module/Microsoft/-ms-flex-pack) and the second one is a legit bug (this very bug).

**Comment by pdehaan** [comment]
If we want to get _really_ silly, we could integrate [stylestats](https://github.com/t32k/stylestats) and then have Grunt generate a report on the CSS after linting (but this is less useful if we bundle normalize.css into the single CSS file since it will include those stats). Here's a sample report from [**grunt-stylestats**](https://github.com/tvooo/grunt-stylestats) (with the normalize.css lines commented out):

```sh
$ grunt stylestats
Running "stylestats:app" (stylestats) task

public/styles/compiled.css
 Published                         Mon Feb 09 2015 13:21:16 GMT-0800 (…
 Paths                             public/styles/compiled.css
 Stylesheets                       1
 Size                              9.4KB
 Data Uri Size                     0
 Rules                             76
 Selectors                         93
 Simplicity                        81.72%
 Most Identifers                   3
 Most Identifers Selector          #user-info .menu-chunk + .menu-chunk
 Lowest Cohesion                   14
 Lowest Cohesion Selector          #global-header #search-box input
 Total Unique Font Sizes           5
 Unique Font Size                  12px
                                   14px
                                   15px
                                   16px
                                   24px
 Total Unique Colors               8
 Unique Color                      #0095DD
                                   #00FF00
                                   #333333
                                   #4D4D4D
                                   #666666
                                   #FF0000
                                   #FFFFFF
                                   INHERIT
 Id Selectors                      30
 Universal Selectors               3
 Unqualified Attribute Selectors   0
 Javascript Specific Selectors     0
 Important Keywords                0
 Float Properties                  3
 Media Queries                     6
 Properties Count                  display: 16
                                   color: 13
                                   width: 12
                                   height: 11
                                   margin: 9
                                   background-image: 8
                                   padding: 8
                                   font-size: 7
                                   -webkit-flex: 7
                                   -ms-flex: 7
Done, without errors.
```

Not sure that those color stats are accurate though... But I think we do have about 6 media queries, 5 different font sizes, 8-ish colors...

**Comment by pdehaan** [comment]
Fixed by #313.
I may file a separate "needs discussion" bug for the **scss-lint** and **grunt-contrib-csslint** tasks. On one hand, I don't want to bloat the package.json with needless devDependencies, but on the other hand the tasks are borderline useful.
